POEApi.Model/JSONProxy/Stash.cs

```diff
@@ -26,6 +26,14 @@ public class Requirement
         public int DisplayMode { get; set; }
     }
 
+    public class IncubatedDetails
```

Comment:
Why not name it `IncubatorDetails`?

Reply:
You apply the Incubator to the item you have equipped. Thus making it the "Details" which have been "Incubated"